CONTRIBUTING.md

```diff
@@ -1 +1,84 @@
-# Contributing
+# How to contribute
+
+One of the easiest ways to contribute is to participate in discussions and discuss issues. You can also contribute by submitting pull requests with code changes.
+
+## General feedback and discussions?
+Please start a discussion on the [Home repo issue tracker](https://github.com/nicolastakashi/flepper/issues).
+
+## Filing issues
+When filing issues, please use our [issue templates](https://github.com/nicolastakashi/flepper/blob/contributing-md/ISSUE_TEMPLATE.MD).
+The best way to get your bug fixed is to be as detailed as you can be about the problem.
+Providing a minimal project with steps to reproduce the problem is ideal.
+Here are questions you can answer before you file a bug to make sure you're not missing any important information.
+
+1. Did you read the [documentation](https://github.com/nicolastakashi/flepper/wiki)?
+2. Did you include the snippet of broken code in the issue?
+3. What are the *EXACT* steps to reproduce this problem?
+4. What package versions are you using (you can see these in the `.csproj` file)?
+5. What operating system are you using?
+
+GitHub supports [markdown](https://help.github.com/articles/github-flavored-markdown/), so when filing bugs make sure you check the formatting before clicking submit.
+
+## Contributing code and content
+Make sure you can build the code. Familiarize yourself with the project workflow and our coding conventions. If you don't know what a pull request is read this article: https://help.github.com/articles/using-pull-requests.
+
+Before submitting a feature or substantial code contribution please discuss it with the team and ensure it follows the product roadmap. You might also read these two blogs posts on contributing code: [Open Source Contribution Etiquette](http://tirania.org/blog/archive/2010/Dec-31.html) by Miguel de Icaza and [Don't "Push" Your Pull Requests](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/) by Ilya Grigorik. Note that all code submissions will be rigorously reviewed and tested by the Flepper team, and only those that meet an extremely high bar for both quality and design/roadmap appropriateness will be merged into the source.
+
+**Commit/Pull Request Format**
+
+```
+Summary of the changes (Less than 80 chars)
+ - Detail 1
+ - Detail 2
+
+Addresses #bugnumber (in this specific format)
+```
+
+**Tests**
+
+-  Tests need to be provided for every bug/feature that is completed.
+-  If there is a scenario that is far too hard to test there does not need to be a test for it.
+- "Too hard" is determined by the team as a whole.
+
+## Contribution Workflow
+
+You can contribute to .NET Core with issues and PRs. Simply filing issues for problems you encounter is a great way to contribute. Contributing implementations is greatly appreciated.
```

Comment:
.NET Core?

Reply:
I'm sorry, I relied on other repositories, it ended up rolling a confusion